src/CommunityToolkit.Maui.Core/Interfaces/IGravatarImageSource.shared.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace CommunityToolkit.Maui.Core;
```

Comment:
I don't think we need this interface as we don't want developers providing an implementation.

Reply:
Agreed

Reply:
Interface removed and appropriate implementations made.